src/features/DashboardPanel/DashboardPanel.tsx

```diff
@@ -95,6 +96,7 @@ interface DashboardPanelViewsProps {
     navigateHomeRefetch: () => void;
     refetch: () => void;
     refetchCount: () => void;
+    user: User;
```

Comment:
Pass in user through props in order to be able to update the context when using the updateUser mutation, also used for displaying the user information on the profile page